### PR TITLE
Create Tilemap and Character movement

### PR DESCRIPTION
**Pull request by ioannisbalo**

### CODE REVIEW

File: Assets/Scripts/GameController.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    [SerializeField] CharacterController characterController;
+
+    void Start()
+    {
+        
+    }
+
+    void Update()
+    {
+        if (Input.GetMouseButtonUp(0))
```
Comment (ptziallas) [inline Assets/Scripts/GameController.cs:18]: why GetMouseButtonUp instead of GetMouseButtonDown?

File: Assets/Scripts/TilemapController.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Tilemaps;
+
+public class TilemapController : MonoBehaviour
+{
+    [SerializeField] Tilemap tilemap;
```
Comment (ptziallas) [inline Assets/Scripts/TilemapController.cs:8]: Tilemap ?

File: Assets/Scripts/TilemapController.cs [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    {
+
+    }
+
+    void Update()
+    {
+        
+    }
+
+    public TileBase GetTile(Vector3Int position)
```
Comment (ptziallas) [inline Assets/Scripts/TilemapController.cs:20]: TileBase?

**Review by ptziallas — APPROVED** [review]

File: Assets/Scripts/GameController.cs (hunk 1 above)
Comment (ioannisbalo) [reply Assets/Scripts/GameController.cs:18]: GetMouseButtonUp is triggered only one time, GetMouseButtonDown can be triggered on many consecutive frames.

File: Assets/Scripts/TilemapController.cs (hunk 2 above)
Comment (ioannisbalo) [reply Assets/Scripts/TilemapController.cs:8]: Unity's structure for managing grid based environments.

File: Assets/Scripts/TilemapController.cs (hunk 3 above)
Comment (ioannisbalo) [reply Assets/Scripts/TilemapController.cs:20]: Individual Tile, Unity name again.

**Pull request merged**